pages/star-search/index.tsx

```diff
@@ -178,7 +179,7 @@ function StarSearchWidget({ widgetDefinition }: { widgetDefinition: WidgetDefini
   );
 }
 
-export default function StarSearchPage({ userId, ogImageUrl }: StarSearchPageProps) {
+export default function StarSearchPage({ userId: initialUserId, ogImageUrl }: StarSearchPageProps) {
```

Comment:
No need to have an `initialUserId`. We can remove the server-side prop for the userId and manage it only on the client-side.